### PR TITLE
Correcting some spelling errors

### PR DESCRIPTION
**Pull request by ymlong**
Correcting some spelling errors

### CODE REVIEW

**Comment by xichengliudui** [comment]
@ymlong  Thank you for your contribution.
/ok-to-test
/kind cleanup
/priority backlog
/release-note-none

File: pkg/apis/authentication/types.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ type TokenRequest struct {
 // TokenRequestSpec contains client provided parameters of a token request.
 type TokenRequestSpec struct {
 	// Audiences are the intendend audiences of the token. A recipient of a
-	// token must identitfy themself with an identifier in the list of
```
Comment (liggitt) [inline pkg/apis/authentication/types.go]: this also exists in the external type and in the generated docs... can we fix it globally?

**Pull request merged**